packages/babel-parser/test/fixtures/typescript/export/export-type-from/input.ts

```diff
@@ -0,0 +1 @@
+export type { T } from './mod';
```

Comment:
Can you add a new test
```ts
export type from "./mod"
```
where both `exportDefaultFrom` and `typescript` are enabled? So we can cover the situation when `this.isContextual("type")` is `true` but the lookahead char is not left bracket.